charts/opentelemetry-collector/templates/NOTES.txt

```diff
@@ -17,3 +17,7 @@
 {{- if not .Values.configMap.create }}
 [WARNING] "configMap" wil not be created and "config" will not take effect.
 {{ end }}
+
+{{- if not (eq (toString .Values.containerLogs) "<nil>") }}
+{{ fail "[ERROR] 'containerLogs' should not be used.  Use 'presets.logsCollection' instead. See https://github.com/open-telemetry/opentelemetry-helm-charts/blob/main/charts/opentelemetry-collector/UPGRADING.md#0231-to-0240 for instructions on how to migrate." }}
```

Comment:
Will this ever be triggered? My feeling is that the schema validation will fail first. If that's the case, I would go with a backward compatible approach.

Reply:
Oh you kept the deprecated `containerLogs` option in the schema. Makes sense now.

Reply:
Still why not make it backward compatible for some time? Shouldn't be that complicated, right?

Reply:
I think we could.  I had originally not allowed it bc of the override stuff, but I think now that we can update the list and are using `| uniq` it should be ok.  I'll update that.

Reply:
Done.  `containerLogs.enabled` functions like `presets.logsCollection.enabled`.  If customers are using a config similar to the previous logs-collection example, their experience should not change.